bin/pygrb/pycbc_pygrb_plot_injs_results

```diff
@@ -21,26 +21,17 @@
 Plot found/missed injection properties for the triggered search (PyGRB).'
 """
 
-# =============================================================================
-# Preamble
-# =============================================================================
-import sys
-import os
-import logging
+import h5py, numpy, logging, os.path, argparse, sys
 import matplotlib.pyplot as plt
-from matplotlib import rc
-import numpy as np
```

Comment:
If you leave this as `import numpy as np`, fewer lines will be highlighted as modified and parsing the changes will be easier.